src/helpers.ts

```diff
@@ -0,0 +1,21 @@
+export function permute(list: any[]) {
```

Comment:
Permute is a non-obvious function.

Please document its functionality, parameters and return value

Also make sure to add a typescript return type

```
/**
 * Description here
 * @param list - Description of list here
 * @returns - What the function returns
 */